### PR TITLE
Add prow.sh script

### PR DESCRIPTION
**Pull request by TerryHowe**
This script will set up the credentials for Prow.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: development/kops/prow.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+fi
+
+BASEDIR=$(dirname "$0")
+
+cat << EOF > ${BASEDIR}/config
+[default]
+output = json
+region = us-west-2
+role_arn=$AWS_ROLE_ARN
+web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
```
Comment (TerryHowe) [inline development/kops/prow.sh:28]: I don't know the name of the secret. That probably isn't correct.

File: development/kops/prow.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+cat << EOF > ${BASEDIR}/config
+[default]
+output=json
+region=us-west-2
+role_arn=$PROW_ROLE_ARN
+web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
+
+[profile conformance-test]
+role_arn=$TEST_ROLE_ARN
+region=us-west-2
```
Comment (micahhausler) [inline development/kops/prow.sh:37]: Can this be parameterized?

**Review by micahhausler — CHANGES_REQUESTED** [review]

File: development/kops/prow.sh (hunk 2 above)
Comment (TerryHowe) [reply development/kops/prow.sh:37]: The region?

Comment (micahhausler) [reply development/kops/prow.sh:37]: Yes. If kops is running in `us-east-1` wouldn't you want to set that here?

Comment (TerryHowe) [reply development/kops/prow.sh]: Latest change allows overriding default of us-west-2 with AWS_REGION or AWS_DEFAULT_REGION

Comment (vivek-koppuru) [reply development/kops/prow.sh]: So the region would be specified in the prowjob itself, and the customer would do that by setting the environment variable before running the script? I think it's okay to have us-west-2 as default as this is specifically for our prow job, but the other options is to actually pass in the region as a string to this script.

Comment (TerryHowe) [reply development/kops/prow.sh]: Done

File: development/kops/prow.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+cat << EOF > ${BASEDIR}/config
+[default]
+output = json
+region = us-west-2
+role_arn=$AWS_ROLE_ARN
+web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
+
+[profile conformance-test]
+role_arn = arn:aws:iam::379412251201:role/DocsDeploymentRole
```
Comment (bnrjee) [inline development/kops/prow.sh]: Should we use ArtifactsDeploymentRole instead?

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**